New pages: HTMLMeterElement properties (#35638)
mdn/content: files/en-us/web/api/htmlmeterelement/low/index.md

---
title: "HTMLMeterElement: low property"
short-title: low
slug: Web/API/HTMLMeterElement/low
page-type: web-api-instance-property
browser-compat: api.HTMLMeterElement.low
---

{{ APIRef("HTML DOM") }}

> The **`low`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the low boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`low`](/en-US/docs/Web/HTML/Element/meter#low) attribute, or the value of `min` if not defined. The value of `low` is clamped by the `min` and `max` values.

Reviewer: ```suggestion
The **`low`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the low boundary of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`low`](/en-US/docs/Web/HTML/Element/meter#low) attribute, or the value of `min` if not defined. The value of `low` is clamped by the `min` and `max` values.
```

This property can also be set directly, for example to set a default value based on some condition.

## Value

> A number. The value of `low`, but not less than {{DOMxRef("HTMLMeterElement.min")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.

Reviewer: ```suggestion
A number that is not less than {{DOMxRef("HTMLMeterElement.min")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.
```
